src/entity.rs

```diff
@@ -422,6 +422,14 @@ impl<C: Config> Entity<C> {
     ///
     /// [interact event]: crate::client::ClientEvent::InteractWithEntity
     pub fn hitbox(&self) -> Aabb<f64> {
+        fn baby(is_baby: bool, adult_hitbox: [f64; 3]) -> [f64; 3] {
```

Comment:
Might even be able to use a closure here. It's even shorter because of the type inference.
```rust
let baby = |is_baby, adult_hitbox| if is_baby { adult_hitbox.map(|a| a / 2.0) } else { adult_hitbox };
```
Although the Rust compiler might complain about missing types. If that's the case then the function is fine. Doesn't really matter.